src/providers/overlayManager.tsx

```diff
@@ -10,7 +10,7 @@ type OverlayManagerContextValue = {
   scrollLocked: boolean
   setOverlaysSet: React.Dispatch<React.SetStateAction<Set<string>>>
   dialogContainerRef: React.RefObject<HTMLDivElement>
-  contextMenuContainerRef: React.RefObject<HTMLDivElement>
+  contextContainerRef: React.RefObject<HTMLDivElement>
```

Comment:
Do we still need to go through portal and overlay manager? I think `popper` handles this automatically? After all we currently have working tooltips that don't have a separate ref in overlay manager

Reply:
Switched to tippy, it has some different abstractions and handles more things on its own but animations might be a bit trickier here and that might or might not matter

Reply:
Seems this was removed from the `ContextMenu`, do we still need this for anything?